docs/tails_printing_guide.rst

```diff
@@ -7,12 +7,75 @@ potentially untrusted network. This is why **we strongly recommend using
 a printer that does not have WiFi or Bluetooth**, and hooking up to it
 using a regular USB cable to print.
 
-Normally, any printer should work with Tails "out of the box." Most
-difficulties stem from not selecting the right driver (extra software
-needed for the printer and computer to communicate). Luckily, Tails has
-a large number of drivers for just about any popularly manufactured
-printer on hand, without even having to download new drivers from the
-web.
+Finding a printer that works with Tails can be challenging because Tails is
+based on the Linux operating system, which often has second-class hardware
+support in comparison to operating systems such as Windows or macOS.
+
+We `maintain a list of printers
+<https://docs.securedrop.org/en/stable/hardware.html#printers>`_ that we have
+personally tested and gotten to work with Tails, in the Hardware guide; if
+possible, we recommend using one of those printers. The Linux Foundation also
+maintains the `OpenPrinting database <https://www.openprinting.org/printers>`_,
+which documents the compatibility, or lack thereof, of numerous printers from
+most every manufacturer.
+
+.. note:: The latest generations of printers might or might not be represented
+          by the OpenPrinting database; also, the database does not document
+          whether or not a printer is wireless, so this will involve manually
+          checking models of interest, if you wish to use this resource as a
+          guide for purchasing a non-wireless printer suitable for use with
+          SecureDrop.
+
+With that in mind, this database is arguably the best resource for researching
+the compatibility of printers with Linux. As a tip for narrowing down your
+search, look for printers that are compatible with Debian, or Debian-based
+distributions like Ubuntu, since Tails itself is also Debian-based. This might
+increase the chances for a successfully seamless installation experience in
+Tails.
+
+In any case, this document outlines the usual set of steps that we follow when
+attempting to use a new printer with Tails, and provides some troubleshooting
+tips that you may find useful if you are trying to use a different printer.
+
+Installing and Printing via the Tails GUI
+-----------------------------------------
+
+Let's look at Tails 3.0's typical flow for installing a USB-connected printer.
+
+First, connect the printer to your Tails-booted computer via USB, then turn the
+printer on.
```

Comment:
According to @harlo, this may have changed in Tails 3. It seems admin privileges were required to add a printer in Tails 1.x, but they are no longer required in Tails 3.x. That would seem to be confirmed by this [resolved Tails issue](https://labs.riseup.net/code/issues/8443)

@justintroutman Can you test and confirm that Tails 3 does not need an admin passphrase to add a printer? If that is the case, then you should also update the Troubleshooting section of this document to remove the now-outdated and incorrect recommendations to set an admin password upon boot in order to print.